test-utils/src/commonTest/kotlin/TestBlockingAndCancelScopeTest.kt

```diff
@@ -0,0 +1,45 @@
+/*
+ Copyright 2021 Splendo Consulting B.V. The Netherlands
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+ */
+
+package com.splendo.kaluga.test
+
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.launch
+import kotlin.test.Test
+import kotlin.test.assertEquals
+import kotlin.time.seconds
+
+class TestBlockingAndCancelScopeTest {
+
+    @Test
+    fun testCanceling(): Unit = testBlockingAndCancelScope {
```

Comment:
Something to consider: I prefer to give tests names that more explicitly describe the expected behaviour. For instance in this case it could be "it_cancels_the_scope"  or "it_doesnt_hangs" (can in camel cases or a combination of snake and camel case of course). In this case, there is no need for comments